src/50_Samples_%26_Templates/Comment_Section.html

```diff
@@ -0,0 +1,144 @@
+
+    <style amp-custom>
+    .full-width {
+      width: 100%;
+    }
+    form.amp-form-submit-success [submit-success] {
+        width: 100%;
+        color: black;
+    }
+    form.amp-form-submit-success .data-input,
+    form.amp-form-submit-success .img-data-input, 
+    form.amp-form-submit-success .button{
+        display: none;
+    }
+    form.amp-form-submit-error [submit-error] {
+        color: red;
+    }
+    .date {
+      color: #a8a3ae;
+      font-size: 12px;
+    }
+    .comment {
+        background: #fff;
+        margin: 0 0 16px 0;
+        width:100%;
+    }
+    .nomargin {
+        margin: 0;
+    }
+    .space-left {
+      margin-left: 16px;
+    }
+    .padding-top {
+      padding-top: 16px;
+    }
+    .padding-left {
+      padding-left: 16px;
+    }
+    .comment-user {
+      display: flex;
+    }
+    </style>
+
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script id="amp-access" type="application/json">
+    {
+        "authorization": "<%host%>/samples_templates/[[.Path]]/authorization?rid=READER_ID&url=CANONICAL_URL&ref=DOCUMENT_REFERRER&_=RANDOM",
+        "pingback": "<%host%>/samples_templates/comment_section/pingback?rid=READER_ID&ref=DOCUMENT_REFERRER&url=CANONICAL_URL",
+        "login": {
+          "sign-in": "<%host%>/samples_templates/[[.Path]]/login?rid=READER_ID&url=CANONICAL_URL",
+          "sign-out": "<%host%>/samples_templates/[[.Path]]/logout?returnUrl=AMPDOC_URL"
+        },
+        "authorizationFallbackResponse": {
+            "error": true,
+            "access": false
+        }
+    }
+    </script>
+    </head>
+    <body>
+    <!-- #### Add a Comment Section -->
+  <!--
+    Use the `amp-access` to realize a login flow. After clicking on the `login` button, you can use the user `test-user` to perform a successfull login. You can then logout by clicking on the `logout` button. Learn more about amp-access [here](/components/amp-access). 
+    Use the `amp-form` component to realize a comment section. Learn more about amp-form [here](/components/amp-form).
+  -->
+  <div>
+    <h5 class="padding-top">Comments</h5>
```

Comment:
padding-top and padding-left are bad names for css rules. -> wrong level of abstraction